src/vs/base/common/async.ts

```diff
@@ -35,6 +35,7 @@ export function createCancelablePromise<T>(callback: (token: CancellationToken)
 	});
 
 	return new class implements CancelablePromise<T> {
+		[Symbol.toStringTag]!: string;
```

Comment:
We still support IE and there is no support for `Symbol`

Reply:
Without an initializer, this is (sortof) a type only position - it makes the checker happy when node pulls in the lib with symbol, but doesn't actually affect emit.